CIP-0XXX/README.md

```diff
@@ -0,0 +1,561 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
+Status: Proposed
+Authors:
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - ?
```

Comment:
I'd say you should be first author @jpraynaud

We may also include @abailly as he wrote the original draft (if he wants)?

```suggestion
    - Sebastian Nagel <sebastian.nagel@ncoding.at>
```

Reply:
Done, I added @abailly and @ch1bo in the authors list :+1: